cpu/esp8266/esp-now/esp_now_netif.c

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
Also maybe use someting like `esp_now_gnrc`, because it isn't the netif part that is of interest here (the implementation below wouldn't work with `lwip_netif` e.g.).

Reply:
This should be changed in #10581 first.